src/lib.rs

```diff
@@ -161,6 +161,15 @@ impl<'a> BitReader<'a> {
         Ok(value)
     }
 
+    /// Read a single bit as a boolean value.
+    /// Interprets 1 as true and 0 as false.
+    pub fn read_bool(&mut self) -> Result<bool> {
+        match try!(self.read_value(1, 8)) {
```

Comment:
While it doesn't matter technically, the second parameter should probably be just 1, as the maximum number of bits read here is always just 1. (The second parameter is used in read_value() only for validation purposes.)